website/static/js/pages/registration-edit-page.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+require('css/registrations.css');
+
+var ko = require('knockout');
+var $ = require('jquery');
```

Comment:
This require statement may be unnecessary.
https://github.com/abought/osf.io/blob/develop/webpack.common.config.js#L148